packages/react-client/src/__tests__/ReactFlight-test.js

```diff
@@ -512,6 +512,59 @@ describe('ReactFlight', () => {
     );
   });
 
+  describe('Hooks', () => {
```

Comment:
Do we have a test for how `useId` would work for sever components that have children that are client components and need the same ID?

Reply:
I don't know if I'm misunderstanding some finer point but my expectation is that if you use `useId` in Flight you will if necessary pass that id as props to client components like any other prop when needed for some purpose

Reply:
Yeah, just checking if there's a test for it already 👍